xero_files.yaml

```diff
@@ -683,7 +685,7 @@ paths:
             example: '{
                         "Name": "My Docs"
                       }'  
-  /Folders/{FolderId}:
+/Folders/{FolderId}:
```

Comment:
Invalid indentation. please indent the endpoints at the same level like the previous endpoints.

Reply:
This is done  at the same level like the previous endpoints.